### ISSUE TITLE
Wildcard ignore

### ISSUE THREAD

**Issue by hashar** [PyCQA/pydocstyle#72]
flake8 and pep8 interprets values passed to 'ignore' as prefix to ignore. So one can:

```
pep8 --ignore=W
```

That will ignore any pep8 ignore.

I wanted to have pep257 to ignore any missing docstrings (D10{1,2,3,4}) by using:

```
pep257 --ignore=D10
```

But that would only ignore D10 errors and let D101 and others flow in.

pep8 handles it via startswith():

```
 def ignore_code(self, code)
 "check if code should be ignored"
    return (code.startswith(self.options.ignore) and
        not code.startswith(self.options.select))
```

**Comment by farmersez** [comment]
In my opinion it should be done explicitly, maybe with an asterisk? something like ``--ignore=D10*``?

**Comment by Nurdok** [comment]
I think we should support an API similar to `pep8` / `flake8`.

**Comment by varunagrawal** [comment]
Addressed in #210 since this is a minor change.

**Comment by varunagrawal** [comment]
Addressed in #212
